objects.js

```diff
@@ -4875,6 +4894,41 @@ StageMorph.prototype.penTrailsMorph = function () {
     return morph;
 };
 
+StageMorph.prototype.addLayer = function(canvas, name) {
+    var layer = {
+        canvas: canvas,
+        name: name };
+
+    if (!this.getLayer(name)) {
+        this.layers.push(layer)
+    }
+
+    return layer;
+};
+
+StageMorph.prototype.getLayer = function(name) {
```

Comment:
Actually I thought the new policy was to use ``for`` iterators, but I'm happy to hear it's not! :)